x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_details.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { usePageId } from '../use_page_id';
+import { usePolicyDetailsSelector } from './policy_hooks';
+import { selectPolicyDetails } from '../../store/policy_details/selectors';
+
+export const PolicyDetails = React.memo(() => {
+  usePageId('policyDetailsPage');
```

Comment:
I don't think we are using this anymore (@peluja1012 right?).
I know Policy List has it, but the PR I was working (currently on hold) removes use of this hook in favor of  `userNavigagtedToPage` store action that is dispatched every time the URL changes.

Reply:
removed, following a similar method as management and alerts lists now